src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/BasicCoercion.txt

```diff
@@ -46,7 +46,7 @@ true
 
 // Erroneous Date -> Bool coercion
 >> Date(2000,1,2) && 1
-Errors: Error 0-14: Invalid argument type. Expecting one of the following: Boolean, Number, Text, OptionSetValue.
+Errors: Error 0-14: Invalid argument type. Expecting one of the following: Boolean, Number, Text, OptionSetValue, UntypedObject.
```

Comment:
>UntypedObject [](http://example.com/codeflow?start=114&length=13)

Just noticed that, in public documentation, we reference this as 'Untyped object'.

Reply:
In all our binder error messages we use the DType enum name